src/app/modules/bounce/__tests__/bounce-test-helpers.ts

```diff
@@ -26,6 +26,7 @@ const makeEmailNotification = (
   formId: ObjectId,
   submissionId: ObjectId,
   recipientList: string[],
+  emailType: 'Admin (response)' | 'Email confirmation',
```

Comment:
nit: Make this into an enum? Since it's used more than once :P

Reply:
can use the enum declared in `mail.ts` constants file